libqrencode.yaml

```diff
@@ -0,0 +1,62 @@
+package:
+  name: libqrencode
+  version: 4.1.1
+  epoch: 0
+  description: Library for encoding QR codes
+  copyright:
+    - license: LGPL-2.1-or-later
+
+environment:
+  contents:
+    packages:
+      - autoconf
+      - automake
+      - autotools-dev
+      - libtool
+      - pkg-config
+      - libpng-dev
+
+pipeline:
+  - uses: git-checkout
+    with:
+      repository: https://github.com/fukuchi/libqrencode
+      tag: ${{package.version}}
+      expected-commit: 715e29fd4cd71b6e452ae0f4e36d917b43122ce8
+
+  - uses: autoconf/configure
+
+  - uses: autoconf/make
+
+  - uses: autoconf/make-install
+
+  - uses: strip
+
+subpackages:
+  - name: libqrencode-dev
```

Comment:
for dev-packages, we generally add the main package as the runtime dependency and use test/pkgconf pipeline to test it out. 

An example can be found here: 
https://github.com/wolfi-dev/os/blob/5407aedeeffda5d5b69bdeb9ebae13129f77600e/libyang.yaml#L42-L52